app/client/components/pages/dashboardPage.jsx

```diff
@@ -160,6 +160,11 @@ class DashboardPage extends React.Component {
               "Active Participants",
               "Number of unique participants who have sent or received a transfer"
             ],
+            [
+              "total_population_cumulative",
+              "Cumulative Participants",
+              "Cumulative number of new participants created"
+            ],
```

Comment:
As far as I understand, I think Sandra was requesting Total users, not Cumulative.

Reply:
I was thinking they were the same, just because I get total users at any given time through accumulating daily users